### PR TITLE
update kafo to 0.6.12 (DEB)

### PR DESCRIPTION
**Pull request by mmoll**
...and some other changes to get more lintian clean.
As there were no code changes, this is safe for 1.9, too

### CODE REVIEW

**Comment by domcleal** [comment]
Thanks, mergfed as fb44751 to deb/develop and building now.  Is there a need for 1.9?

**Comment by mmoll** [comment]
No real need, but it would ease things for people using Debian/testing

**Comment by domcleal** [comment]
Okay, merged as 690d030.

File: dependencies/trusty/kafo/control [hunk 1]
```diff
@@ -2,20 +2,25 @@ Source: ruby-kafo
...
 XB-Ruby-Versions: ${ruby:Versions}
-Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter,
-         puppet (< 4.0.0),
-         ruby-clamp,
-         ruby-highline (< 1.7),
+Depends: ruby | ruby-interpreter,
+         puppet (<< 4.0.0),
+         ruby-clamp (>= 0.6.2),
+         ruby-highline (<< 1.7),
+         ruby-highline (>= 1.6.21),
```
Comment (domcleal) [inline dependencies/trusty/kafo/control]: This has caused a problem on trusty, as it ships .20: http://packages.ubuntu.com/trusty/ruby-highline.  We build our own for wheezy and precise, while jessie has .21.

http://ci.theforeman.org/job/systest_foreman/7272/console

**Pull request closed without merging**